src/activation_cuda.cu

```diff
@@ -653,6 +653,84 @@ std::unique_ptr<BaseLayer> SoftmaxCuda::to_host()
     return host_layer;
 }
 
+////////////////////////////////////////////////////////////////////////////////
+/// AGVI
+////////////////////////////////////////////////////////////////////////////////
+AGVICuda::AGVICuda() {}
```

Comment:
I believe this `exp` activation and not AGVI approach itself. I would call it `ExpCuda`

Reply:
@lhnguyen102 About the name, this exponential activation function is only applied for the V2_bar states (or odd states in this implementation). Shouldn't we call it then something different to differentiate it from an exponential activation function that applies to all hidden states? Or should we find a way to manage to which states we apply the exp act. func.?

Reply:
Oh I see. It would make sense to call it something different then

Reply:
What about ExpV2 or ExpAGVI?

Reply:
`EvenExp` or `PairExp`?